rpcs3/Emu/RSX/Capture/rsx_replay.cpp

```diff
@@ -15,7 +15,7 @@ namespace rsx
 	{
 		// 'fake' initialize usermemory
 		// todo: seriously, need to probly watch the replay memory map and just make sure its mapped before we copy rather than do this
-		user_mem_addr = vm::falloc(vm::get(vm::user1m)->addr, 0x10000000);
+		user_mem_addr = vm::falloc(vm::get(vm::user64k)->addr, 0x10000000);
```

Comment:
I've changed it specifically to 1m to be compatible with potential sys_rsx accuracy improvements in the future.

Reply:
1m does not work as the vm block is not default-initialized (nullptr deref). You did not test the changes, they broke capture replay.

Reply:
ahh, I see. its missing the new parameter of area_size, which allocates the block if it doesnt exist.

Reply:
Will remove before merge, need to keep it for testers for now.